build.gradle.kts

```diff
@@ -22,8 +20,8 @@ plugins {
     id("com.github.johnrengelman.shadow") version "4.0.3"
     id("com.google.osdetector") version "1.4.0"
     id("org.ajoberstar.grgit") version "2.0.0" apply false
-    id("net.ltgt.errorprone") version "0.0.16"
-    id("com.github.spotbugs") version "1.6.4"
+    //id("net.ltgt.errorprone") version "0.0.16"
+    //id("com.github.spotbugs") version "1.6.4"
```

Comment:
Spotbugs doesn't work on 11?

Reply:
Updating the Gradle plugin seems to do the trick... However, there are several new false positives